README.md

```diff
@@ -68,23 +68,19 @@ must be applied via [patches](patches). See [below](#patches) for more info on p
 1. After the PR is merged to update a component, coordination is often needed in the darc dependency flows. The source-build-external update
 may need to flow in at the same time as the cooresponding changes in product repos which take a dependency on the new component version.
 
-### Updating an External Component Used in a Toolset Repo and in a Target Repo
+### Updating an External Component Used in a Pre-SBE Repo
```

Comment:
What do you think about explaining what Pre-SBE means?  I think this will be confusing to the read so it needs to be explicitly defined in the first sentence or two of this section.